requirements-test.txt

```diff
@@ -1,8 +1,22 @@
-tox==2.3.1 \
```

Comment:
This file used to be our pre-tox one, but I've repurposed it and moved the requirements that we used to specify in tox.ini into here, and pinned them. This should help with reproducability.